source/js/dropdowns.js

```diff
@@ -0,0 +1,27 @@
+const Dropdowns = {
+  init: function () {
+    // Toggle dropdowns when dropdown is clicked
+    const dropdowns = document.querySelectorAll('.dropdown-toggle');
+    for (let i = 0; i < dropdowns.length; i++) {
+      const element = dropdowns[i];
+      element.addEventListener('click', function(e) {
+        e.preventDefault()
+        element.classList.toggle("show")
+        const id = element.id
+        const target = document.querySelector(`[aria-labelledby="${id}"]`)
+        target.classList.toggle("d-block")
+      })
+    }
```

Comment:
Handy bit of API: `querySelectorAll` returns an array-like that does not offer `map`, but _does_ offer `forEach`:

```suggestion
    document.querySelectorAll(`.dropdown-toggle`).forEach(element =>
      element.addEventListener(`click`,  _ => {
        element.classList.toggle("show");
        const target = document.querySelector(`[aria-labelledby="${element.id}"]`);
        target.classList.toggle("d-block");
      })
    );
```

Also note that if we use proper `<button>` markup, there is no default behaviour that needs preventing.